records/config.go

```diff
@@ -74,6 +74,10 @@ type Config struct {
 	ExternalOn bool
 	// EnforceRFC952 will enforce an older, more strict set of rules for DNS labels
 	EnforceRFC952 bool
+	// SetTruncateBit when `false` ensures responses never have the Truncate bit set even
+	// if they were truncated. When `true` any message that gets truncated will have the
+	// Truncate bit set.
```

Comment:
Perhaps a comment here about setting the value to `false` results in non-compliance, but is required for some clients that are non-compliant?